crates/but/src/id/tests.rs

```diff
@@ -225,6 +244,40 @@ fn non_commit_ids_do_not_collide() -> anyhow::Result<()> {
         },
     ]
     "#);
+
+    // uncommitted hunks do not collide with anything else
```

Comment:
Something I am still struggling with is the lack of a traversal over all available ShortIDs, so one has to somehow know the Id to request.
I might look into a test-tool like that.